app/views/public_recipes/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<section class="public_recipe_container">
+  <div class="row">
+    <% if @recipes.empty? %>
+      <h2>You don't have any existing recipe of your own!</h2>
+    <% else %>
+      <article class="row d-flex border p-3 mb-5 bg-white">
+        <% @recipes.each do |recipe| %>
+          <%= render partial: "recipe", locals: { recipe:, recipe_foods: @recipe_foods } %>
+        <% end %>
+      </article>
+    <% end %>
+  </div>
+</section>
```

Comment:
- [ ] I am unable to visit this page due to an error. Kindly fix it.

  ![xcv-public-recipe](https://user-images.githubusercontent.com/36542986/175530288-36dde81f-49a7-43e6-86c4-1ae4fb19d235.jpg)

Reply:
@TedLivist Thanks for pointing that out, I tested on my end and it works without issues I am not sure what caused that.

Reply:
I tested with both scenarios when the list is empty and when it's not empty and they are both working correctly.

Reply:
It might've been an issue related to the N+1 query that I found there, I fixed it hopefully this error won't happen again.